include/php_driver.h

```diff
@@ -46,6 +46,7 @@ extern "C"
 #include <Zend/zend_types.h>
 
 #define LL_FORMAT "%lld"
+#define L_FORMAT "%ld"
```

Comment:
I think we can completely remove ```L_FORMAT```, there is ```PRId64``` found in ```#include <inttypes.h>```, it would be great to remove ```#define L_FROMAT```